.github/workflows/run_tests.yml

```diff
@@ -235,12 +235,9 @@ jobs:
       - name: Install go-swagger
         id: install-go-swagger
         run: |
-          sudo apt-get update
-          sudo apt-get install -y apt-transport-https gnupg curl
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/gpg.2F8CB673971B5C9E.key' | sudo apt-key add -
-          curl -1sLf 'https://dl.cloudsmith.io/public/go-swagger/go-swagger/config.deb.txt?distro=debian&codename=any-version' | sudo tee /etc/apt/sources.list.d/go-swagger-go-swagger.list
-          sudo apt-get update
-          sudo apt-get install -y swagger
+          cd /usr/local/bin
+          wget -o swagger https://github.com/go-swagger/go-swagger/releases/download/v0.29.0/swagger_linux_amd64
```

Comment:
The command line option to put the result in `swagger` is `-O` (capital O), not lowercase o.